src/client/common/components/base-network-view/base-network-view.js

```diff
@@ -208,7 +213,8 @@ class BaseNetworkView extends React.Component {
       h('div.graph', {
           className: classNames({
             'graph-network-loading': this.state.networkLoading,
-            'graph-sidebar-open': this.state.open
+            'graph-sidebar-open': this.state.open && this.state.activeMenu != 'filter-menu',
```

Comment:
Could you play around with trying to inject the width as a prop?

e.g
```js
h('div.graph', {
  className: classNames(... graph-sidebar-open: this.state.open),
  style: { width: this.state.sidebarWidth }
})
```

Reply:
- [ ] create a default width if no width is specified in the props
- [ ] in your interactions feature, inject the special width into the base view component